gateway-js/src/datasources/__tests__/RemoteGraphQLDataSource.test.ts

```diff
@@ -1,5 +1,5 @@
 import { fetch } from '../../__mocks__/apollo-server-env';
-import { makeFetchHappenFetcher} from '../../__mocks__/make-fetch-happen-fetcher';
+import { makeFetchHappenFetcher } from '../../__mocks__/make-fetch-happen-fetcher';
```

Comment:
There are those who would complain about applying prettier to a file, but not me.

Reply:
This is actually a result of using inline snapshots (which has the effect of prettifying the file when a new snap is printed). I ended up not using a snapshot but the changes persisted. I don't mind either - I'd usually isolate to a commit but hopefully this one was small enough to not be a bother.